bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/io/RetryListener.java

```diff
@@ -62,10 +64,11 @@ public void onHeaders(Metadata headers) {
   @Override
   public void onClose(Status status, Metadata trailers) {
     if (isRetriableStatus(status.getCode())
-        && isRetriableCall
         && !stateSignalledToListener) {
       if (retryingCall.retryCallAfterBackoff(payload, requestHeaders, this)) {
-        // We are retrying
+        // A retryable error.
+        failedCount += 1;
+        LOG.info("Retrying failed call. Failure #" + failedCount + ", got: " + status, status.getCause());
```

Comment:
This should probably be a Logger.warn.  If the getCause is a throwable, that should be the first argument.